.github/workflows/stale.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 */6 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          # General rules applied to both, issues and pull requests (PRs)
+          start-date: "2022-07-30T00:00:00Z"
+          days-before-close: 14
+          days-before-stale: 90
+          exempt-all-assignees: true
+          exempt-all-milestones: true
+
+          # Rules applied to Issues
+          exempt-issue-labels: "Epic, bug, priority-critical, priority-high, community-upvoted, for-review, security, keep"
```

Comment:
Are these case sensitive? I'd probably add clarifications (hopefully no clarification takes more than 90 days) and v3.

Reply:
I believe they are and yes, I will need to adapt them to our cases. Thanks for noticing

Reply:
Adapted to our situation.